cypress/components/confirm/confirm.cy.tsx

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable jest/valid-expect */
+/* eslint-disable no-unused-expressions */
```

Comment:
Could be one line:
/* eslint-disable jest/valid-expect , no-unused-expressions */

Reply:
Don't think this will effect the tests, but noted and will make changes in next PR